regression/contracts/invar_dynamic_struct_member/main.c

```diff
@@ -0,0 +1,30 @@
+typedef struct test
+{
+  int x;
+} test;
+
+void main()
+{
+  struct test *t = malloc(sizeof(test));
+  t->x = 0;
+
+  unsigned n;
+  for(unsigned i = 0; i < n; ++i)
+    __CPROVER_loop_invariant(i <= n)
+    {
+      switch(i % 4)
+      {
+      case 0:
+        break;
+
+      case 1:
+      case 2:
+        t->x += 1;
```

Comment:
Is this meant to fall through

Reply:
Yes, I wanted to keep one fall-through case in this `switch`.

Reply:
May be worth adding a comment `// fall through`